src/components/common/CopyButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconButton } from '@rmwc/icon-button';
+import React from 'react';
+
+interface CopyButtonProps {
+  text: string;
+  label?: string;
+}
+
+// TODO(davideast): Depend on pkg https://github.com/jaredLunde/react-hook/tree/master/packages/copy#readme
+function useCopy(text: string) {
+  const [copied, setCopied] = React.useState(false);
+  const reset = React.useRef(() => setCopied(false));
+  React.useEffect(() => reset.current, [text]);
+
+  return {
+    copied,
+    copy: React.useCallback(
+      () =>
+        navigator.clipboard
+          .writeText(text)
+          .then(() => setCopied(true))
+          .catch(() => setCopied(copied => copied)),
```

Comment:
Seems like the type of copied is boolean, but you're setting a callback here, is this intended?